src/catchup/ApplyLedgerChainWork.cpp

```diff
@@ -264,6 +265,11 @@ ApplyLedgerChainWork::onRun()
         CLOG(ERROR, "History") << "Replay failed";
         throw;
     }
+    catch (FileSystemException&)
+    {
+        CLOG(ERROR, "History") << POSSIBLY_CORRUPTED_LOCAL_FS;
```

Comment:
I think you need to also log the reason `e.what()` when you catch those exceptions?

Reply:
It looks like `XDRInputFileStream` already logs the message.